resources/istio/charts/kiali/templates/virtualservice.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: kiali-virtualservice
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ template "kiali.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+spec:
+  gateways:
+  - kyma-gateway.kyma-system.svc.cluster.local
+  hosts:
+  - "kiali.kyma.local"
```

Comment:
this should be parametrized - on a cluster you have different domain. You can refere to `{{ .Values.global.domainName }}`